tests/GltfTests.cpp

```diff
@@ -0,0 +1,150 @@
+#include "CesiumGltfReader/GltfReader.h"
+#include "testUtils.h"
+
+#include "cesium/omniverse/GltfAccessors.h"
+#include "cesium/omniverse/GltfUtil.h"
+
+#include <CesiumGltf/Material.h>
+#include <CesiumGltf/MeshPrimitive.h>
+#include <CesiumGltf/Model.h>
+#include <doctest/doctest.h>
+#include <sys/types.h>
+
+#include <cstdio>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <optional>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <gsl/span>
+#include <yaml-cpp/yaml.h>
+
+const std::string ASSET_DIR = "tests/testAssets/gltfs";
+const std::string CONFIG_PATH = "tests/configs/gltfConfig.yaml";
+
+// simplifies casting when comparing some material queries to expected output from config
+bool operator==(pxr::GfVec3f v3, std::vector<float> v) {
+    return v.size() == 3 && v3[0] == v[0] && v3[1] == v[1] && v3[2] == v[2];
+}
+
+TEST_SUITE("gltf utils") {
+    TEST_CASE("IndicesAccessor smoke test") {
+        uint64_t data;
+        std::list<uint64_t> dataContainer = {42, 64, 8675309, 21};
+
+        DOCTEST_VALUE_PARAMETERIZED_DATA(data, dataContainer);
+
+        CHECK(cesium::omniverse::IndicesAccessor(data).size() == data);
+    }
+
+    void checkGltfExpectedResults(const std::filesystem::path& gltfFileName, YAML::Node expectedResults) {
+
+        // --- Load Gltf ---
+        std::ifstream gltfStream(gltfFileName, std::ifstream::binary);
+        std::stringstream gltfBuf;
+        gltfBuf << gltfStream.rdbuf();
+        CesiumGltfReader::GltfReader reader;
+        CesiumGltfReader::GltfReaderResult gltf =
+            reader.readGltf(gsl::span(reinterpret_cast<const std::byte*>(gltfBuf.str().c_str()), gltfBuf.str().size()));
+
+        if (!gltf.errors.empty()) {
+            for (const std::string& err : gltf.errors) {
+                std::cerr << err;
+            }
+            throw std::runtime_error("failed to parse model");
+        }
+
+        // gltf.model is a std::optional<CesiumGltf::Model>, make sure it exists
+        if (!(gltf.model && gltf.model->meshes.size() > 0)) {
+            throw std::runtime_error("test model is empty");
+        }
+
+        // --- Begin checks ---
+        CesiumGltf::MeshPrimitive prim = gltf.model->meshes[0].primitives[0];
+        CesiumGltf::Model model = *gltf.model;
+
+        namespace gltfUtil = cesium::omniverse::GltfUtil;
```

Comment:
I think it would be cleaner to add `using namespace cesium::omniverse;` to the top of the file (just under the includes). Then you should be able to use `GltfUtil::`.